internal/releaseversion/releaseversion.go

```diff
@@ -12,21 +12,30 @@ import (
 	"github.com/giantswarm/azure-admission-controller/internal/errors"
 )
 
+const (
+	ignoreReleaseAnnotation = "release.giantswarm.io/ignore"
```

Comment:
Maybe copy this to `apiextensions` as well?

Reply:
yeah, I just wanted to run the idea with you folks first and try it out :)